scripts/monitoring/cron-send-usage-pv.py

```diff
@@ -121,14 +121,15 @@ def main():
     if dynamic_pod_name == "":
         pass
     else:
-        MAXIMUM_CLUSTER_CAPACITY = get_max_capacity(dynamic_pod_name)
-        logger.debug(MAXIMUM_CLUSTER_CAPACITY)
+        cluster_capacity_max = get_max_capacity(dynamic_pod_name)
+        logger.debug(cluster_capacity_max)
```

Comment:
seeing this in the logs has no context .. 

maybe `logger.debug("cluster_capacity_max: %s", cluster_capacity_max)` ??